docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1712,14 +1712,33 @@ Both `Meter` and `DirectMeter` messages share the following fields:
   extern instance.
 
 * `spec`, a message of type `MeterSpec` used to describe the capabilities of
-  this meter extern instance. Currently, the `MeterSpec` message is used to
-  carry only the meter unit, which can be any of the `MeterSpec.Unit` enum
-  values:
+  this meter extern instance. The `MeterSpec` message is used to describe the
+  meter unit and the meter type. The meter unit can be any of the
+  `MeterSpec.Unit` enum values:
     * `UNSPECIFIED`: reserved value.
     * `BYTES`, which signifies that this meter can be configured with rates
       expressed in bytes/second.
     * `PACKETS`, for rates expressed in packets/second.
 
+  The meter type can be any of the `MeterSpec.Type` enum values:
+    * `TWO_RATE_THREE_COLOR`: This is the *Two Rate Three Color Marker* (trTCM)
+      defined in RFC 2698 [@RFC2698]. This is the standard P4Runtime meter type
+      and allows meters to use two rates to split packets into three potential
+      colors: GREEN, YELLOW, or RED. This mode is the default, but can also be
+      set explicitly in a P4 program by adding the `@two_rate_three_color`
+      annotation to the meter definition.
+    * `SINGLE_RATE_THREE_COLOR`: This is the *Single Rate Three Color Marker*
+      (srTCM) defined in RFC 2697 [@RFC2697]. This allows meters to use one rate
+      and an Excess Burst Size (EBS) to split packets into three potential
+      colors: GREEN, YELLOW, or RED. In a P4 program, this mode can be set by
+      adding the `@single_rate_three_color` annotation to the meter definition.
+    * `SINGLE_RATE_TWO_COLOR`: This is a simplified version of RFC 2697
+      [@RFC2697], which restricts meters to use only a single rate to mark
```

Comment:
Can we explicitly say which single rate?

Reply:
Yupp, added.